### ISSUE TITLE
Wrong amount of Specs on the output

### ISSUE THREAD

**Issue by alladywek** [kotest/kotest-gradle-plugin#12]
Hi, I started to investigate this plugin and faced with an interesting thing, I have minimal project setup and found that Gradle produces the wrong amount of Specs in results.
I have only 1 test file:
```
package testingstyles

import io.kotest.core.spec.style.StringSpec
import io.kotest.data.forAll
import io.kotest.data.row
import io.kotest.matchers.shouldBe
import kotlin.math.max

class StringSpecExampleTest : StringSpec() {
    init {
        "maximum of two numbers" {
            forAll(
                row(1, 5, 5),
                row(1, 0, 1),
                row(0, 0, 0)
            ) { a, b, max ->
                max(a, b) shouldBe max
            }
        }
    }
}
```
and build script:
```
plugins {
    kotlin("jvm") version "1.4.31"
    id("io.kotest") version "0.2.6"
}

group = "org.example"
version = "0.0.1"

val kotestVersion = "4.4.3"

repositories {
    mavenCentral()
}

tasks.withType<Test> {
    useJUnitPlatform()
}

dependencies {
    implementation(kotlin("stdlib"))
    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3")
    testImplementation("io.kotest:kotest-assertions-core:$kotestVersion")
    testImplementation("io.kotest:kotest-framework-engine-jvm:$kotestVersion")
}

```
if you run Gradle command for tests ` ./gradlew clean kotest`, than output is: 
```
...
>> Kotest
- Feeding the kotest engine with freshly harvested tests
- Test plan has 2 specs

1.  io.kotest.core.script.ScriptSpec

2.  testingstyles.StringSpecExampleTest
    - maximum of two numbers OK

>> All tests passed

Specs:   2 passed, 0 failed, 2 total
Tests:   1 passed, 0 failed, 0 ignored, 1 total
Time:    0s

BUILD SUCCESSFUL in 2s
3 actionable tasks: 3 executed
```
The first test is from kotest itself `1.  io.kotest.core.script.ScriptSpec` but it should not be.
Tested with Gradle wrapper version: 6.7 and 6.8.3

**Comment by rezammalik** [comment]
I actually noticed this as well.

**Comment by sksamuel** [comment]
ScriptSpec is a bug that crept into 4.4 and will be fixed in the 4.5 for
April.

**Comment by sksamuel** [comment]
See https://github.com/kotest/kotest/issues/2288
